ddtrace/sampler.py

```diff
@@ -108,9 +101,15 @@ def set_sample_rate(self, sample_rate):
         self.sample_rate = float(sample_rate)
         self.sampling_id_threshold = self.sample_rate * _MAX_UINT_64BITS
 
-    def sample(self, span):
-        # type: (Span) -> bool
-        return ((span._trace_id_64bits * KNUTH_FACTOR) % _MAX_UINT_64BITS) <= self.sampling_id_threshold
+    def sample(self, span, allow_false=False):
+        # type: (Span, bool) -> bool
+        sampled = ((span._trace_id_64bits * KNUTH_FACTOR) % _MAX_UINT_64BITS) <= self.sampling_id_threshold
+        # NB allow_false has weird functionality here, doing something other than "allowing false" to be returned
```

Comment:
what does NB mean?

Reply:
"please pay attention to this" https://www.quora.com/What-does-the-abbreviation-N-B-mean-and-why-do-people-use-it/answer/Robert-Charles-Lee